requirements/common.txt

```diff
@@ -9,7 +9,7 @@ Flask-JWT>=0.3.2,<0.4
 requests-oauthlib>=0.7.0,<1
 icalendar>=3.11,<4
 requests[security]>=2.12.4,<3
-psycopg2>=2.6.2,<2.7
+psycopg2>=2.6.1
```

Comment:
Please explain the reason for this change. `<2.7` was intentional as we did not want a breaking update.

Reply:
@niranjan94  i google the log ,why travis failing so i find out this happen when version 2.7 is already there so i removed the <2.7 ,and added 2.6.1 so the version which is after 2.6.1 will get added automatically but i forgot that removing upper limit can cause `breaking update ` problem, I will wait till Travis problem gets fixed and will open a new PR will add your name and email address